diff -up libgssglue-0.1/configure.in.save libgssglue-0.1/configure.in
--- libgssglue-0.1/configure.in.save	2011-03-31 11:32:43.724306000 -0400
+++ libgssglue-0.1/configure.in	2011-03-31 11:33:32.249745000 -0400
@@ -6,6 +6,7 @@ AC_INIT(libgssglue, 0.1, nfsv4-wg@citi.u
 AC_CONFIG_SRCDIR([src/g_accept_sec_context.c])
 AM_INIT_AUTOMAKE
 AM_PROG_LIBTOOL
+AC_CONFIG_MACRO_DIR([m4])
 
 AC_CONFIG_HEADERS([src/gssglue/config.h])
 
@@ -25,8 +26,8 @@ AC_TYPE_SIZE_T
 AC_TYPE_UID_T
 AC_CHECK_SIZEOF(short)
 AC_CHECK_SIZEOF(int)
-AC_CHECK_SIZEOF(long)
-AC_CHECK_SIZEOF(size_t)
+#AC_CHECK_SIZEOF(long)
+#AC_CHECK_SIZEOF(size_t)
 
 # Checks for library functions.
 AC_FUNC_MALLOC
